frog/frog.scrbl

```diff
@@ -984,3 +868,152 @@ In your Frog project directory, create an output directory named
+Should @italic{not} end in trailing slash. Used to form full URIs for
+various purposes including @tt{urn:}s in feeds and the
+@tt|{@full-uri}| variable supplied to @secref["templates"].}
+
+@defparam[current-uri-prefix v (or/c string? #f) #:value #f]{A path
+prepended to URIs, including those specified here in @secref["config"]
+such as @tt{permalink} and @tt{posts-index-uri}. Changing this from
+the default @racket["/"] is useful when you want to embed your blog in
+another web site.}
+
+@defparam[current-editor v string? #:value "$EDITOR"]{What editor to
+launch with @tt{raco frog @literal{--}edit}. @racket["$EDITOR"] means
+to use the @tt{$EDITOR} environment variable.}
+
+@defparam[current-editor-command v string? #:value "{editor} {filename}"]{
+The command to run, in case you need to customize how the editor is
+called. For example, @racket["{editor} {filename}"] will do
+@racket[(system "$EDITOR 2012-01-01-a-blog-post.md")]. See the test
+submodule in @tt{paths.rkt} for more examples.}
+
+@defparam[current-show-tag-counts? v boolean? #:value #t]{Whether to
+show the count of posts next to each tag in the
+@secref["page-template"] variable @tt{tags/feeds}.}
+
+@defparam[current-permalink v string? #:value "/{year}/{month}/{title}.html"]{
+Pattern for blog post permalinks. An example of the Jekyll "pretty"
+style would be
+@racket["/blog/{year}/{month}/{day}/{title}/index.html"].
+
+Another available pattern is @tt{{filename}}, which is the
+@tt{this-part} portion of your post's @tt{YYYY-MM-DD-this-part.md}
+file name. This is in case you don't like Frog's encoding of your post
+title and want to specify it exactly yourself, e.g. to match a
+previous blog URI.}
+
+@defparam[current-posts-index-uri v string? #:value "/index.html"]{
+URI of the first index page for posts.}
+
+@defparam[current-index-full? v boolean? #:value #t]{Should index page
+items contain full posts -- more than just the portion above ``the
+jump'' @litchar{<!-- more -->} marker (if any)?}
+
+@defparam[current-index-newest-first? v boolean? #:value #t]{Should
+index page items be sorted newest first?}
+
+@defparam[current-feed-full? v boolean? #:value #t]{Should feed items
+contain full posts -- more than just the portion above ``the jump''
+@litchar{<!-- more -->} marker (if any)?}
+
+@defparam[current-posts-per-page n exact-positive-integer? #:value 10]{
+How many posts per page for index pages?}
+
+@defparam[current-max-feed-items n exact-positive-integer? #:value 20]{
+How many items to include in feeds? Older items in excess of this will
+not appear in the feed at all.}
+
+@defparam[current-decorate-feed-uris? v boolean? #:value #t]{Decorate
+feed URIs with Google Analytics query parameters like
+@tt{utm_source}?}
+
+@defparam[current-feed-image-bugs? v boolean? #:value #t]{Insert in
+each feed item an image bug whose URI is decorated with Google
+Analytics query parameters like @tt{utm_source}?}
```

Comment:
Although I like to document a boolean as the question it answers, it's good to know it can be confusing. I'll rephrase.